sycl/doc/EnvironmentVariables.md

```diff
@@ -77,6 +55,49 @@ Assuming a filter has all three elements of the triple, it selects only those de
 
 Note that all device selectors will throw an exception if the filtered list of devices does not include a device that satisfies the selector. For instance, `SYCL_DEVICE_FILTER=cpu,level_zero` will cause `host_selector()` to throw an exception. `SYCL_DEVICE_FILTER` also limits loading only specified plugins into the SYCL RT. In particular, `SYCL_DEVICE_FILTER=level_zero` will cause the `cpu_selector` to throw an exception since SYCL RT will only load the `level_zero` backend which does not support any CPU devices at this time. When multiple devices satisfy the filter (e..g, `SYCL_DEVICE_FILTER=gpu`), only one of them will be selected.
 
+## Controlling DPC++ Level Zero Plugin
+
+| Environment variable | Values | Description |
+| -------------------- | ------ | ----------- |
+| `SYCL_ENABLE_PCI` | Integer | When set to 1, enables obtaining the GPU PCI address when using the Level Zero backend. The default is 0. |
+| `SYCL_PI_LEVEL_ZERO_DISABLE_USM_ALLOCATOR` | Any(\*) | Disable USM allocator in Level Zero plugin (each memory request will go directly to Level Zero runtime) |
+| `SYCL_PI_LEVEL_ZERO_TRACK_INDIRECT_ACCESS_MEMORY` | Any(\*) | Enable support of the kernels with indirect access and corresponding deferred release of memory allocations in the Level Zero plugin. |
+
+`(*) Note: Any means this environment variable is effective when set to any non-null value.`
+
+## Tools variables
+
+| Environment variable | Values | Description |
+| -------------------- | ------ | ----------- |
+| `INTEL_ENABLE_OFFLOAD_ANNOTATIONS` | Any(\*) | Enables ITT Annotations support for SYCL runtime. This variable should only be used by tools, that support ITT Annotations. |
```

Comment:
Does it make sense to mention xpti related vars here?

Reply:
Not sure. They all come from different libraries. E.g. XPTI_FRAMEWORK_DISPATCHER comes from libxpti.a, and the rest of the variables comes from whatever library was specified in XPTI_FRAMEWORK_DISPATCHER (e.g. XPTI_TRACE_ENABLED is defined in libxptifw.so).

Reply:
I can still mention them and add a note, that these are only available under certain circumstances.

Reply:
Since we are linking xpti static library into libsycl.so, I believe we need to mention env vars which this library reads. Probably a link to xpti documentation would be OK.